storage/api/Storage.Samples/SetClientEndpoint.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_set_client_endpoint]
+
+using System;
+using Google.Cloud.Storage.V1;
+
+public class SetClientEndpoint

```

Comment:
Please follow the naming and styling as in https://github.com/GoogleCloudPlatform/dotnet-docs-samples/tree/main/kms/api

```suggestion
public class SetClientEndpointSample

```